source/extensions/filters/common/local_ratelimit/local_ratelimit_impl.cc

```diff
@@ -48,11 +48,12 @@ LocalRateLimiterImpl::LocalRateLimiterImpl(
       throw EnvoyException(
           "local rate descriptor limit is not a multiple of token bucket fill timer");
     }
+    // Save the multiplicative factor to control the descriptor refill frequency.
     new_descriptor.multiplier_ = token_bucket.fill_interval_ / token_bucket_.fill_interval_;
```

Comment:
Sorry I meant the local in this function to make it more clear what this match is doing.

```suggestion
    new_descriptor.multiplier_ = per_descriptor_token_bucket.fill_interval_ / token_bucket_.fill_interval_;
```

/wait